reference/configuration/framework.rst

```diff
@@ -1446,6 +1462,120 @@ value.
     For more information, see
     :ref:`component-serializer-converting-property-names-when-serializing-and-deserializing`.
 
+.. _reference-cache:
+
+cache
+~~~~~
+
+.. _reference-cache-app:
+
+app
+...
+
+**type**: ``string`` **default**: ``cache.adapter.filesystem``
+
+The cache adapter used by the ``cache.app`` service.
+
+.. note::
+
+    The framework bundle ships with multiple adapters: apcu, doctrine, system,
+    filesystem, psr6, and redis.
+
+system
+......
+
+**type**: ``string`` **default**: ``cache.adapter.system``
+
+The cache adapter used by the ``cache.system`` service.
+
+directory
+.........
+
+**type**: ``string`` **default**: ``%kernel.cache_dir%/pools``
+
+The path to the cache directory used by services inheriting from the
+``cache.adapter.filesystem`` adapter (including ``cache.app``).
+
+default_doctrine_provider
+.........................
+
+**type**: ``string``
+
+The service name to use as your default Doctrine provider.
+
+default_psr6_provider
+.....................
+
+**type**: ``string`` **default**: ``%kernel.cache_dir%/pools``
+
+The service name to use as your default PSR-6 provider.
+
+default_redis_provider
+......................
+
+**type**: ``string`` **default**: ``redis://localhost``
+
+The dsn to use by the Redis provider.
+
+pools
+.....
+
+**type**: ``array``
+
+A list of cache pools to be created by the framework extension.
+
+.. _reference-cache-pools-name:
+
+name
+""""
+
+**type**: ``prototype``
+
+Name of the pool you want to create.
+
+.. note::
+
+    Your pool name must differ from ``cache.app`` or ``cache.system``.
+
+adapter
+#######
+
+**type**: ``string`` **default**: ``cache.app``
+
+A cache adapter.
```

Comment:
This description should be improved to explain the purpose of this option. Thanks!